test/box/fix-memory-corruption-server.lua

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env tarantool
+
+require('console').listen(os.getenv('ADMIN'))
+
+box.cfg({
+    listen = 3301,
```

Comment:
You can't choose just random constant ports. Test run works in parallel, the port may be already occupied by some other test. You need to check how other tests start and how do they specify the `listen` parameter.

Currently your test fails if I run it like this: `python test-run.py box/fix-memory-corruption box/fix-memory-corruption box/fix-memory-corruption box/fix-memory-corruption box/fix-memory-corruption `. Which is totally legal and is used quite often to find flaky tests.

Reply:
fixed